client/state/data-layer/wpcom/activity/index.js

```diff
@@ -0,0 +1,45 @@
+/**
```

Comment:
To follow convention and reflect the endpoint route, this should probably be moved under `sites`.